src/main/java/com/github/lunatrius/laserlevel/marker/Marker.java

```diff
@@ -18,6 +19,7 @@ public Marker(final BlockPos pos, final int spacing, final int rgb) {
         this.enabled = true;
         this.spacing = spacing;
         this.rgb = rgb;
+        this.maxdistance = maxdistance;
```

Comment:
Typo? It doesn't do anything - no maxdistance parameter, so it's equal to `this.maxdistance = this.maxdistance;`.

Can set it to 64 on this line instead.